graphql/introspection/testdata/TestComputeSchemaJSON.snapshots.json

```diff
@@ -56,10 +56,12 @@
               "name": "skip"
             },
             {
-                "args": [],
-                "description": "Client-side-only directive that instructs the type generator to mark this field as optional. This is useful for making the generated types compliant with Troy persistence schema.",
-                "locations": ["FIELD"],
-                "name": "type_as_optional"
+              "args": [],
+              "description": "Client-side-only directive that instructs the type generator to mark this field as optional. This is useful for making the generated types compliant with Troy persistence schema.",
+              "locations": [
+                "FIELD"
+              ],
```

Comment:
@CodeBrew28 this diff gets generated for me, unclear why. I can revert it if it fails things.

Reply:
Hm, it seems like just the formatting changed, which seems okay. Not sure why it was wrong before.